nirc_ehr/src/org/labkey/nirc_ehr/dataentry/section/NIRCGrossPathologyFormSection.java

```diff
@@ -0,0 +1,45 @@
+package org.labkey.nirc_ehr.dataentry.section;
+
+import org.labkey.api.view.template.ClientDependency;
+
+import java.util.List;
+
+public class NIRCGrossPathologyFormSection extends BaseFormSection
```

Comment:
You need a trigger script for this dataset just with `require("ehr/triggers").initScript(this);`. To ensure QCState gets set correctly.

Reply:
done